src/Illuminate/Notifications/Console/NotificationTableCommand.php

```diff
@@ -5,15 +5,22 @@
 use Illuminate\Console\MigrationGeneratorCommand;
 use Symfony\Component\Console\Attribute\AsCommand;
 
-#[AsCommand(name: 'notifications:table')]
+#[AsCommand(name: 'make:notifications-table')]
 class NotificationTableCommand extends MigrationGeneratorCommand
 {
     /**
      * The console command name.
      *
      * @var string
      */
-    protected $name = 'notifications:table';
+    protected $name = 'make:notifications-table';
+
+    /**
+     * The console command name aliases.
+     *
+     * @var array
+     */
+    protected $aliases = ['notifications:table'];
```

Comment:
@nunomaduro agreed

Reply:
Hmm on second thought this could maybe not hurt. People have been used to these command names for a while. Maybe it's best we support them indefinitely?